corehq/apps/case_search/utils.py

```diff
@@ -182,11 +183,14 @@ def build_query(self, search_criteria, commcare_sort=None):
         return search_es
 
     def _get_initial_search_es(self):
+        max_results = CASE_SEARCH_MAX_RESULTS
+        if toggles.INCREASED_MAX_SEARCH_RESULTS.enabled(self.request_domain):
+            max_results = 1000
```

Comment:
I feel like this should got in the const.py next to `CASE_SEARCH_MAX_RESULTS`

Reply:
it has to be runtime, though, since it varies by domain.  You could make a `get_case_search_max_results(domain)` function and put it over there, I suppose.